test/parallel/test-whatwg-url-searchparams-constructor.js

```diff
@@ -16,23 +16,24 @@ assert.strictEqual(params + '', 'a=b');
 params = new URLSearchParams(params);
 assert.strictEqual(params + '', 'a=b');
 
-// URLSearchParams constructor, empty.
+// URLSearchParams constructor, no arguments
+params = new URLSearchParams();
```

Comment:
Not yet, nor do we have a `new URLSearchParams(undefined)` test. Both are added.